system-variables.md

```diff
@@ -1932,8 +1936,12 @@ explain select * from t where age=5;
     - For uncommitted read-only transactions, you can commit the transactions normally.
     - For uncommitted transactions that are not read-only, SQL statements that perform write operations in these transactions are rejected.
     - For uncommitted read-only transactions with modified data, the commit of these transactions is rejected.
+- `tidb_restricted_read_only` affects [`tidb_super_read_only`](#tidb_super_read_only-new-in-v531) in some cases:
+    - Setting `tidb_restricted_read_only` to `ON` will let [`tidb_super_read_only`](#tidb_super_read_only-new-in-v531) to `ON`.
+    - Setting `tidb_restricted_read_only` to `OFF` leaves [`tidb_super_read_only`](#tidb_super_read_only-new-in-v531) unchanged.
+    - If `tidb_restricted_read_only` is `ON`, [`tidb_super_read_only`](#tidb_super_read_only-new-in-v531) cannot be set to `OFF`.
 - After the read-only mode is enabled, all users (including the users with the `SUPER` privilege) cannot execute the SQL statements that might write data unless the user is explicitly granted the `RESTRICTED_REPLICA_WRITER_ADMIN` privilege.
-- Users with `RESTRICTED_VARIABLES_ADMIN` or `SUPER` privileges can modify this variable. However, if the [security enhanced mode](#tidb_enable_enhanced_security) is enabled, only the users with the `RESTRICTED_VARIABLES_ADMIN` privilege can modify this variable.
+- Users with `SUPER` or `SYSTEM_VARIABLES_ADMIN` privilege can modify this variable. However, if the [security enhanced mode](#tidb_enable_enhanced_security) is enabled, the additional `RESTRICTED_VARIABLES_ADMIN` privilege is required to read or modify this variable.
```

Comment:
Technically LGTM. But I'm still concerned: The difference between these 2 variables is the privilege control part, but the user might take some effort to find the difference. As a reference, I took 3 minutes to figure out the intention of making 2 variables. Should we highlight the difference?

Reply:
Yes please. Even a simple sentence like "It is recommended to use `tidb_super_read_only` instead of `tidb_restricted_read_only `."

In my use-case, we are not sure which one to use. I think it is tidb_super_read_only, but I'm not entirely sure.

Reply:
I've added some descriptions for either `tidb_super_read_only` or `tidb_restricted_read_only`. PTAL again, Thank you.